test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/function/DoubleSupplierTest.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.util.function
+
+import java.util.function.DoubleSupplier
+
+import org.junit.Assert._
+import org.junit.Test
+
+class DoubleSupplierTest {
+  import DoubleSupplierTest._
+  private val delta = 1e31
+
+  @Test def getAsDouble(): Unit = {
+    assertEquals(1.234D, makeSupplier(1.234D).getAsDouble(), 0.0)
```

Comment:
The double suffix should be lowercase (or not there, since double is the default):
```suggestion
    assertEquals(1.234, makeSupplier(1.234).getAsDouble(), 0.0)
```